src/plots/plots.js

```diff
@@ -132,32 +132,12 @@ plots.registerSubplot = function(_module) {
     var plotType = _module.name;
 
     if(subplotsRegistry[plotType]) {
-        throw new Error('plot type' + plotType + ' already registered');
-    }
-
-    var attr = _module.attr,
-        idRoot = _module.idRoot;
-
-    var regexStart = '^',
-        regexEnd = '([2-9]|[1-9][0-9]+)?$',
-        hasXY = (plotType === 'cartesian' || subplotsRegistry === 'gl2d');
-
-    function makeRegex(mid) {
-        return new RegExp(regexStart + mid + regexEnd);
+        console.log('plot type ' + plotType + ' already registered');
```

Comment:
It *would*, but there's a check in the `Plotly.register` function that will avoid calling `subplotsRegistry` if the module has already been loaded.